app/views/admin/base/_tree_item.html.haml

```diff
@@ -0,0 +1,10 @@
+%li.sort-tree__item(data-sort-tree-item){id: "list_#{item.id}"}
+  .sort-tree__wrap(data-sort-tree-tolerance)
```

Comment:
Prefer the hash attributes syntax (%tag{ lang: 'en' }) over HTML attributes syntax (%tag(lang=en))